components/TestSubmissionsList/TestSubmissionsList.tsx

```diff
@@ -2,17 +2,39 @@
 import scss from './TestSubmissionsList.module.scss';
 import {TestSubmissionsResults} from "../TestSubmissionsResults/TestSubmissionsResults";
 
+interface TestSubmissionsListProps {
+    tests: string[];
+}
 
-export function TestSubmissionsList(): JSX.Element {
-    
-    //TODO: getTestSubmissions result from API
-    
-    const tests: string [] = ["Test 1", "Test 2", "Test 3"];
-    
+export function TestSubmissionsList(props: TestSubmissionsListProps): JSX.Element {
+
+    const testHolder: string [] = ["Test 1", "Test 2", "Test 3"];
     return (
         <ul className={scss.list}>
             {
-                tests.map(test => <TestSubmissionsResults key={tests.indexOf(test)} test={test}/>)
+                props.tests.length > 0 ?
+                    (props.tests.map(test => {
+                            const testParsed = JSON.parse(JSON.stringify(test));
+                            return (
+                                <TestSubmissionsResults
+                                    key={props.tests.indexOf(test)}
+                                    testId={testParsed.testId}
+                                    testResult={testParsed.testResult}
+                                    testAnswer={testParsed.testAnswer}
+                                />
+                            )
+                        })
+                    ):(testHolder.map(test => {
+                            return (
+                                <TestSubmissionsResults
+                                    key={props.tests.indexOf(test)}
+                                    testId={test}
+                                    testResult={""}
+                                    testAnswer={'Placeholder for Test Results'}
```

Comment:
I'm not sure that we need this - I think its fine for this to be empty if there aren't any tests to display :) 

(we've got an upcoming ticket also which makes it so that there will always be tests! :) )

Reply:
cool. thanks!